spec/controllers/api/v1/subjects_controller_spec.rb

```diff
@@ -387,6 +417,30 @@
             expect(retired).to all( be false )
           end
 
+          describe "favorited subjects" do
```

Comment:
If this is a straight copy of the above, then maybe extract to a shared helper in this spec to avoid the duplication? Up to you

e.g. in that file somewhere
``` ruby
shared_examples "a favorited subject" do
  ...
end

it_behaves_like "a favorited subject"

```